res/values/styles.xml

```diff
@@ -220,6 +220,11 @@
         <item name="android:padding">4dp</item>
     </style>
 
+    <style name="locationIconRounded">
+        <item name="cornerFamily">rounded</item>
+        <item name="cornerSize">50%</item>
```

Comment:
This doesn't compile in AOSP environment. cornerSize expects dp, sp or px

`packages/apps/Etar/res/values/styles.xml:225: error: expected dimension but got (raw string) 50%.`